README.md

```diff
@@ -107,7 +107,7 @@ Many thanks to the ~100 contributors including [@corbpage](https://twitter.com/c
 * [Parasol](https://github.com/Lamarkaz/parasol) - Agile smart contract development environment with testing, INFURA deployment, automatic contract documentation and more. It features a flexible and unopinionated design with unlimited customizability
 * [0xcert](https://github.com/0xcert/framework/) - JavaScript framework for building decentralized applications
 * [OpenZeppelin SDK](https://openzeppelin.com/sdk/) - OpenZeppelin SDK: A suite of tools to help you develop, compile, upgrade, deploy and interact with smart contracts.
-* [sbt-ethereum](https://sbt-ethereum.io/) - A tab-completey, text-based console for smart-contract interaction and development, including wallet and ABI management, ENS support, and advanced Scala integration.
+* [sbt-ethereum](https://sbt-ethereum.io/) - A tab-completely, text-based console for smart-contract interaction and development, including wallet and ABI management, ENS support, and advanced Scala integration.
```

Comment:
Guessing, but otherwise the text makes no sense

```suggestion
* [sbt-ethereum](https://sbt-ethereum.io/) - A tab-completion, text-based console for smart-contract interaction and development, including wallet and ABI management, ENS support, and advanced Scala integration.
```